### ISSUE TITLE
Could not find spec results - empty Summary

### ISSUE THREAD

**Issue by dthisner** [bahmutov/cypress-split#69]
Seeing following in the Cypress run: `Could not find spec results for cypress/e2e/admin/settings.cy.ts`

This is the summary page: 
![image](https://github.com/bahmutov/cypress-split/assets/17165035/167c0637-4d0a-4953-b311-4e66930e2579)

The command that is being run: 
`    "cy:ui": "npx cypress run --env grepTags=regression --spec 'cypress/e2e/**/*'",`

It looks like it is still grabbing the API tests, even when specified the specific spec

My Github action yaml:
```yaml
jobs:
  e2e-ui:
    name: e2e UI Tests
    runs-on: ubuntu-22.04
    strategy:
      fail-fast: false
      matrix:
        containers: [1, 2, 3, 4]
    defaults:
      run:
        working-directory: packages/e2e

    steps:
      - name: Checkout code
        uses: actions/checkout@v3

      - name: Run split Cypress tests 🧪
        uses: cypress-io/github-action@v5
        with:
          command: npm run cy:ui
        env:
          SPLIT: ${{ strategy.job-total }}
          SPLIT_INDEX: ${{ strategy.job-index }}

      - name: Generate Report
        if: failure()
        run: npm run cy:report

      - name: Upload to S3 bucket
        if: failure()
        uses: shallwefootball/s3-upload-action@master
        id: S3
        with:
          aws_key_id: ${{ secrets.AWS_KEY_ID }}
          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
          aws_bucket: ${{ vars.AWS_BUCKET_NAME }}
          source_dir: './packages/e2e/cypress/reports/htmlReport'

      - name: Get Cypress data
        if: failure()
        run: |
          echo TOTAL_TESTS=$(jq '.stats.tests' ./cypress/reports/htmlReport/report.json) >> $GITHUB_ENV
          echo TEST_PASSED=$(jq '.stats.passes' ./cypress/reports/htmlReport/report.json) >> $GITHUB_ENV
          echo TEST_FAILED=$(jq '.stats.failures' ./cypress/reports/htmlReport/report.json) >> $GITHUB_ENV
          echo REPORT_URL="${{ vars.AWS_S3_BUCKET_URL }}/${{steps.S3.outputs.object_key}}/report.html" >> $GITHUB_ENV

      - name: Slack Failure Notification
        if: failure()
        uses: rtCamp/action-slack-notify@v2
        env:
          SLACK_CHANNEL: de5-status-tests
          SLACK_COLOR: failure
          SLACK_MESSAGE: "Tests: ${{ env.TOTAL_TESTS }}\n Passed: ${{ env.TEST_PASSED }}\nFailures: ${{ env.TEST_FAILED }} \n Report: ${{ env.REPORT_URL }}"
          SLACK_TITLE: DEKN Cypress tests failed
          SLACK_USERNAME: cypress-bot
          SLACK_WEBHOOK: ${{ vars.SLACK_WEBHOOK }}
```

**Comment by vergjor** [comment]
@dthisner were you able to resolve this issue? I am facing the same

**Comment by dthisner** [comment]
@vergjor Sadly not :(

**Comment by duckdum** [comment]
I'm facing the same issue. Can someone help? :pray:

**Comment by bahmutov** [comment]
This is due to https://github.com/bahmutov/cypress-split#list-of-specs which has a workaround (or Cypress test runner can implement the necessary change for this plugin to work automatically)

**Comment by duckdum** [comment]
sorry, I could not understand. What's the workaround?

**Comment by bahmutov** [comment]
The workaround is to tell this plugin about the specs you want to run. Whatever the `npm` command uses as `--spec foo/*.cy.js` parameter should be passed via an environment variable so this plugin "knows" which specs to split (and not all Cypress specs)

**Comment by dthisner** [comment]
@duckdum This is how we did it in Github Actions: 
```yaml
dekn-cypress-ui:
    strategy:
      fail-fast: false
      matrix:
        container: [0, 1, 2, 3, 4, 5]
    runs-on: ubuntu-latest
    timeout-minutes: 60
    defaults:
      run:
        working-directory: ./packages/e2e/
    container:
      image: cypress/included:10.11.0
      env:
        TERM: xterm
        FQDN: ${{ inputs.FQDN }}
    steps:
      - uses: actions/checkout@v3
        run: |
          # Find spec files, sort by line count, pick a test every N=job-total
          find cypress/e2e -name "*.cy.ts" -exec wc -l {} \; | sort | cut -d ' ' -f2 | sed -n '${{strategy.job-index}}~${{strategy.job-total}}p' > tests
          cat tests
          echo "TESTS=$(cat tests | paste -sd ',')" >> $GITHUB_ENV
      - run: yarn
        name: Install dependencies
      - run: yarn cypress run --headless --env grepTags=regression --spec $TESTS
        name: Run split Cypress tests 🧪
        id: cypress-test-run
```

So far, it works great! Does not have the nice UI and you end up with X amount of reports at the end

**Comment by juliushaertl** [comment]
I proposed a possible fix for this when using the specPattern cypress config which showed a similar issue at https://github.com/bahmutov/cypress-split/pull/115
